app/components/publications/batch-documents-publication-modal.js

```diff
@@ -72,20 +77,30 @@ export default class PublicationsBatchDocumentsPublicationModalComponent extends
 
   @task
   *saveNewPublication(publicationProperties) {
-    const publicationFlow = yield this.publicationService.createNewPublicationFromMinisterialCouncil(publicationProperties, {
-      case: this.case,
-      agendaItemTreatment: this.agendaItemTreatment,
-      mandatees: this.mandatees,
-    });
+    const publicationFlow =
+      yield this.publicationService.createNewPublicationFromMinisterialCouncil(
+        publicationProperties,
+        {
+          case: this.case,
+          agendaItemTreatment: this.agendaItemTreatment,
+          mandatees: this.mandatees,
+        }
+      );
     this.referenceDocument.publicationFlow = publicationFlow;
     yield this.referenceDocument.save();
+
+    yield this.setRegulationTypeThroughReferenceDocument(
+      this.referenceDocument
+    );
+
```

Comment:
Ben niet echt fan van de PATCH vlak na de POST van publication-flow, we zouden die regulation-type al kunnen weten in de createNewPublication method. De check op het bestaan van regulation-type is overbodig in dat geval.